package.json

```diff
@@ -13,18 +16,22 @@
   "dependencies": {
     "assume": "^1.2.5",
     "aws-sdk": "^2.2.0",
-    "azure-entities": "^0.9.0",
+    "azure-entities": "^0.9.1",
     "babel": "^5.8.21",
```

Comment:
This should probably be removed because it's no longer needed as babel-compile pulls in the correct version of babel-core.